src/components/user/document/AddDocument.jsx

```
@@ -51,99 +51,108 @@ export default function AddDocument({ orderId, userId }) {
+                  type="number"
+                  className="form-control"
+                  id={`userId`}
+                  name={`userId`}
+                  value={userId}
+                  readOnly
+                />
+              </div>
+              <form onSubmit={handleSubmit(onSubmit)}>
+                {fields.map((field, index) => (
+                  <>
+                    <h5>Tài liều thứ {index + 1}</h5>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)